django/contrib/auth/models.py

```diff
@@ -382,14 +382,20 @@ class AbstractUser(AbstractBaseUser, PermissionsMixin):
 
     Username, password and email are required. Other fields are optional.
     """
-    username = models.CharField(_('username'), max_length=30, unique=True,
-        help_text=_('Required. 30 characters or fewer. Letters, digits and '
-                    '@/./+/-/_ only.'),
+    username = models.CharField(
+        _('username'),
+        max_length=30,
+        unique=True,
+        help_text=_(
```

Comment:
this could be a single line - we prefer longer lines up to 119 characters when it helps readability